src/vl-select.js

```diff
@@ -61,6 +61,24 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
     }
   }
 
+  _dataRequiredChangedCallback(oldValue, newValue) {
+    if (newValue && !oldValue) {
+      this.setAttribute('required', '');
+    }
+    if (!newValue && oldValue) {
```

Comment:
kan dit niet in een else block aangezien het het omgekeerde is van bovenstaande conditie (idem methode hieronder)?

Reply:
not(A && B) = not A || not B
dus denk ik niet dat dat het omgekeerde gewoon is?